### DISCUSSION TITLE
FHIRcast testing for OHIF viewer

### DISCUSSION THREAD

**Discussion by mbellehumeur** [medplum/medplum#4580] [General]
Hi,
I am writing a FHIR extension for the OHIF viewer which includes FHIRcast.
The in-progress readme is on my GitHub page.
I would like to test with your server and add an example Medplum config to the documentation.

Thanks,
Martin

**Comment by mbellehumeur** [comment]
I found how to get my app credentials (id, secret).  
I can subscribe, publish, get context.  All good.  I just need to filter out the heartbeat events.

 Anything special I should test? 

If I open a patient in the medplum app, should that generate an patient-open event?

**Reply by mbellehumeur** [reply]
I used the fhircast-demo app from the repo and tested sending 'patient-open' messages across both apps.
The only thing missing is the 'unsubscribe'.  Seems like a low hanging fruit to implement.  Please consider.
I'll add a 'Medplum' section to the OHIF fhir readme.

**Reply by ThatOneBro** [reply]
Hi @mbellehumeur , sorry for the late response to your thread. I just opened https://github.com/medplum/medplum/pull/4590 which should address what you brought up about `unsubscribe` requests. Please let us know if you run into anything else that is blocking you!

**Reply by mbellehumeur** [reply]
Thanks for the unsuscribe!  
My next step would be to start a “IHE IRA worklist actor” project where we would publish DiagnosticReport-open events.
Is there a sample data set with reports that have ImagingStudy resource?
